README.md

```diff
@@ -36,6 +40,49 @@ python edit_app.py
 
 _(For advice on how to get the best results by tuning parameters, see the [Tips](https://github.com/timothybrooks/instruct-pix2pix#tips) section)._
 
+### 🧨 Diffusers
+
+1. Install diffusers and relevant dependencies:
+
+```bash
+pip install transformers accelerate torch
```

Comment:
the code snippet below allows need pillow and requests, should we include that here?

Reply:
It's automatically installed when installing diffusers, so I think we're good :-) 

The other packages are soft dependencies in diffusers that help a lot with speed though and are necessary for this specific model